.travis/maybe-bump-version.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
-if [ "$TRAVIS_NODE_VERSION" != "8.1.2" ]
+if [[ ! "${PUBLISH_NODE_VERSION:-8.1.2}" =~ ^$TRAVIS_NODE_VERSION ]]
 then
-  echo "Skipping version bump for TRAVIS_NODE_VERSION ${TRAVIS_NODE_VERSION}"
+  echo "Skipping pr-bumper for TRAVIS_NODE_VERSION [${TRAVIS_NODE_VERSION}]"
   exit 0
 fi
 
 if [ "$EMBER_TRY_SCENARIO" != "ember-default" ]
 then
-  echo "Skipping version bump for EMBER_TRY_SCENARIO ${EMBER_TRY_SCENARIO}"
+  echo "Skipping pr-bumper for EMBER_TRY_SCENARIO [${EMBER_TRY_SCENARIO}]"
```

Comment:
This line reads differently from: https://github.com/ciena-frost/ember-frost-core/pull/546/files#diff-3aac347fbc380ab603b6957f61185df8R11

Reply:
I know - https://github.com/ciena-frost/ember-frost-core/pull/547 changes it there as well